corehq/apps/hqwebapp/views.py

```diff
@@ -447,6 +448,12 @@ def domain_login(req, domain, custom_template_name=None, extra_context=None):
     return _login(req, domain, custom_template_name, extra_context)
 
 
+def iframe_login(req, domain):
+    return domain_login(req, domain, custom_template_name="hqwebapp/iframe_login.html", extra_context={
+        'current_page': {'page_name': _('Your session has expired')},
+    })
```

Comment:
I see this was already like this, but FYI it looks like `extra_context` is implemented in such a way that doing this would override any other variables in the `'current_page'` dict if one already exists:
```python
In [1]: ctxt = {'foo': 'bar',
   ...:         'current_page': {'page_name': 'name1',
   ...:                          'url': 'example.com'}}

In [2]: ctxt
Out[2]: {'current_page': {'page_name': 'name1', 'url': 'example.com'}, 'foo': 'bar'}

In [3]: ctxt.update({'current_page': {'page_name': 'name2'}})

In [4]: ctxt
Out[4]: {'current_page': {'page_name': 'name2'}, 'foo': 'bar'}  # 'url' is missing
```

Reply:
I'm going to leave it as-is, as the current usages of `extra_context` aren't affected by this, but that is a good point.